command/harness/setup.go

```diff
@@ -102,90 +104,125 @@ func HandleSetup(ctx context.Context, r *SetupVMRequest, s store.StageOwnerStore
 	pools = append(pools, r.PoolID)
 	pools = append(pools, r.FallbackPoolIDs...)
 
-	var poolErr, err error
-	var selectedPool, selectedPoolDriver, owner string
+	var selectedPool, selectedPoolDriver string
+	var poolErr error
 	var instance *types.Instance
 	foundPool := false
 	fallback := false
 
 	st := time.Now()
 
-	// TODO: Remove this once we start populating license information
-	if strings.Contains(r.PoolID, freeAccount) {
-		owner = freeAccount
-	} else {
-		owner = getAccountID(r.Context, r.Tags)
-	}
-
+	// try to provision an instance with fallbacks
 	for idx, p := range pools {
 		if idx > 0 {
 			fallback = true
 		}
 		pool := fetchPool(r.SetupRequest.LogConfig.AccountID, p, env.Dlite.PoolMapByAccount)
 		logr.WithField("pool_id", pool).Traceln("starting the setup process")
```

Comment:
When I run this on local, my code just get stuck. I think it's due to this change `log := logrus.StandardLogger()`. When I reverted that, it worked fine.